tensorflow/core/kernels/mkl_lrn_op.cc

```diff
@@ -108,9 +107,11 @@ class MklLRNOp : public OpKernel {
         return;
       } else if (!src_dnn_shape.IsMklChannelDim(src_dnn_shape.GetDimension() -
                                                 1)) {
-        Tensor converted_tensor =
-            ConvertMklToTF<T>(context, src_tensor, src_dnn_shape);
-        MklDefaultToEigen(context, converted_tensor);
+        Tensor* converted_tensor;
+        OP_REQUIRES_OK(context,
+                       ConvertMklToTF<T>(context, src_tensor, src_dnn_shape,
+                                         converted_tensor));
+        MklDefaultToEigen(context, *converted_tensor);
```

Comment:
I think it's cleaner to make `converted_tensor` a `Tensor` and not `Tensor*`.
```c++
        Tensor converted_tensor;
        OP_REQUIRES_OK(context,
                       ConvertMklToTF<T>(context, src_tensor, src_dnn_shape,
                                         &converted_tensor));
        MklDefaultToEigen(context, converted_tensor);
```

Reply:
Sure, I'll make these changes.